docs/mkdocs.yml

```diff
@@ -92,298 +91,229 @@ plugins:
+      - Prebuilt agents: # NOTE: Moved to under LangGraph to create a hierarchy based by product
+        - agents/overview.md
+        - agents/agents.md
+        - agents/run_agents.md
+        - agents/streaming.md
+        - agents/models.md
+        - agents/tools.md
+        - agents/mcp.md
+        - agents/context.md
+        - agents/memory.md
+        - agents/human-in-the-loop.md
+        - agents/multi-agent.md
+        - agents/evals.md
+        - agents/deployment.md
+        - agents/ui.md
+        - Notebooks: # OPEN QUESTIONS: do we want to keep this? should it be in this section?
+            - how-tos/create-react-agent.ipynb
+            - how-tos/create-react-agent-memory.ipynb
+            - how-tos/create-react-agent-system-prompt.ipynb
+            - how-tos/create-react-agent-hitl.ipynb
+            - how-tos/create-react-agent-structured-output.ipynb
+            - how-tos/create-react-agent-manage-message-history.ipynb
+            - how-tos/react-agent-from-scratch.ipynb
+            - how-tos/react-agent-from-scratch-functional.ipynb
+      - Graph API basics:
+        - how-tos/state-reducers.ipynb
+        - how-tos/sequence.ipynb
+        - how-tos/branching.ipynb
+        - how-tos/recursion-limit.ipynb
+        - how-tos/visualization.ipynb
+      - Controllability:
+        - how-tos/map-reduce.ipynb
+        - how-tos/command.ipynb
+        - how-tos/configuration.ipynb
+        - how-tos/node-retries.ipynb
+        - how-tos/return-when-recursion-limit-hits.ipynb
+      - concepts/breakpoints.md
+      - Persistence:
+        - concepts/persistence.md
+        - concepts/durable_execution.md
+        - how-tos/persistence.ipynb
+        - how-tos/subgraph-persistence.ipynb
+        - how-tos/cross-thread-persistence.ipynb
+        - how-tos/persistence_postgres.ipynb
+        - how-tos/persistence_mongodb.ipynb
+        - how-tos/persistence_redis.ipynb
+        - how-tos/persistence-functional.ipynb
+        - how-tos/cross-thread-persistence-functional.ipynb
+      - Time travel: concepts/time-travel.md
+      - Memory:
+        - concepts/memory.md
+        - how-tos/memory/manage-conversation-history.ipynb
+        - how-tos/memory/delete-messages.ipynb
+        - how-tos/memory/add-summary-conversation-history.ipynb
+        - how-tos/memory/semantic-search.ipynb
+      - Human-in-the-loop:
+        - concepts/human_in_the_loop.md
+        # DEPRECATE - concepts/v0-human-in-the-loop.md
+        - how-tos/human_in_the_loop/breakpoints.ipynb
+        - how-tos/human_in_the_loop/dynamic_breakpoints.ipynb
+        - how-tos/human_in_the_loop/edit-graph-state.ipynb
+        - how-tos/human_in_the_loop/wait-user-input.ipynb
+        - how-tos/human_in_the_loop/time-travel.ipynb
+        - how-tos/human_in_the_loop/review-tool-calls.ipynb
+        - how-tos/wait-user-input-functional.ipynb
+        - how-tos/review-tool-calls-functional.ipynb
+      - Streaming:
```

Comment:
i think we need to bring this one up, perhaps even before persistence. basically need to hammer on key features:

* persistence (ie durable execution)
* human-in-the-loop (enabled by persistence)
* streaming

Reply:
Changed